lib/x11test.pm

```diff
@@ -525,9 +590,22 @@ sub start_firefox {
     assert_screen 'firefox-launch', 90;
 }
 
+sub restart_firefox {
+    my ($self, $cmd) = @_;
+    # exit firefox properly
+    wait_still_screen 2;
+    send_key 'alt-f';
+    assert_screen('firefox-menu-quit');
+    send_key 'q';
+    assert_screen 'xterm';
+    type_string "$cmd\n";
+    type_string "firefox >>firefox.log 2>&1 &\n";
+    $self->firefox_check_default;
+}
+
 sub firefox_check_default {
     # Set firefox as default browser if asked
-    assert_screen [qw(firefox_default_browser firefox_trackinfo firefox_readerview_window firefox_clean)], 150;
+    assert_screen [qw(firefox_default_browser firefox_trackinfo firefox_readerview_window firefox-launch)], 150;
```

Comment:
This does effect more than just the regression tests and maintenance tests. This broke e.g. https://openqa.suse.de/tests/2086883#step/firefox/6 as the "firefox_clean" tag is not looked for anymore. Maybe rename all tags on the existing SLE needles to say "firefox-launch" instead or additional to "firefox_clean"? Also more: https://openqa.suse.de/tests/2086517#step/firefox/7 , https://openqa.suse.de/tests/2086457#step/firefox/5

Reply:
You can’t make an omelette without breaking an egg.

Reply:
I don't want to complain. Sorry if that was misunderstood. I just wanted to make you aware that not only the "regression maintenance tests" are impacted, no hard feelings :)